File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/MailServiceType.scala
package io.swagger.server.model


/**
 * Service type definition for a mail service, including name, pricing, and category.
 *
 * @param services_id The ID of the service type. for example: ''10880''
 * @param services_name The name of the service type. for example: ''MailBaby Mail''
 * @param services_cost The cost of the service type. for example: ''1.00''
 * @param services_category The category of the service type. for example: ''800''
 * @param services_buyable Whether the service type is buyable. for example: ''1''
 * @param services_type The type of the service type. for example: ''800''
 * @param services_field1 Additional field for the service type.
 * @param services_field2 Additional field for the service type.
 * @param services_module The module of the service type. for example: ''mail''
 */
case class MailServiceType (
  services_id: Option[String],
  services_name: Option[String],
  services_cost: Option[String],
  services_category: Option[String],
  services_buyable: Option[String],
  services_type: Option[String],
  services_field1: Option[String],
  services_field2: Option[String],
  services_module: Option[String]
)

File: OPENAPI-REST-API/openapi-client/php-dt/src/App/DTO/MailServiceType.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

/**
 * Service type definition for a mail service, including name, pricing, and category.
 */
class MailServiceType
{
    /**
     * The ID of the service type.
     */
    #[DTA\Data(field: "services_id", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $services_id = null;

    /**
     * The name of the service type.
     */
    #[DTA\Data(field: "services_name", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $services_name = null;

    /**
     * The cost of the service type.
     */
    #[DTA\Data(field: "services_cost", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $services_cost = null;

    /**
     * The category of the service type.
     */
    #[DTA\Data(field: "services_category", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $services_category = null;

    /**
     * Whether the service type is buyable.
     */
    #[DTA\Data(field: "services_buyable", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $services_buyable = null;

    /**
     * The type of the service type.
     */
    #[DTA\Data(field: "services_type", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $services_type = null;

    /**
     * Additional field for the service type.
     */
    #[DTA\Data(field: "services_field1", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $services_field1 = null;

    /**
     * Additional field for the service type.
     */
    #[DTA\Data(field: "services_field2", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $services_field2 = null;

    /**
     * The module of the service type.
     */
    #[DTA\Data(field: "services_module", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $services_module = null;

}
